core/amber/src/main/resources/python_udf/topic_modeling_trainer.py

```diff
@@ -16,7 +16,8 @@ def open(self, *args):
         if len(args) >= 2:
             self._train_args = {"num_topics": int(args[1])}
         else:
-            self._train_args = {"num_topics": 5}
+            self.logger.exception("Not enough arguments in topic modeling operator.")
+            raise RuntimeError("Not enough arguments in topic modeling operator.")
```

Comment:
same here, please remove the duplicate exception log

Reply:
done